### PR TITLE
Minor PlateSpace updates

### PR DESCRIPTION
**Pull request by rlondner**
Removed reference to old repository in package.json
Added small README touch-ups

### CODE REVIEW

File: PlateSpace/Web/README.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ In order to get our Stitch backend-as-a-service application fully set up, we mus
 ### Named pipelines
 
 Stitch lets you define named pipelines.
-Named pipelines are custom pipeline requests that could be executed from the client side / by rules on top of the collections (see Rules section below). To create a named pipeline, select the  `Pipelines` link in the left-hand menu and press the `New` button close to the `Named Pipelines` text.
+Named pipelines are custom pipeline requests that could be executed from the client side or by collection (see [Rules configuration](https://github.com/mongodb/stitch-examples/tree/master/PlateSpace/Web#rules-configuration) below). To create a named pipeline, select the  `Pipelines` link in the left-hand menu and press the `New` button close to the `Named Pipelines` text.
```
Comment (edaniels) [inline PlateSpace/Web/README.md]: could -> can

Comment (edaniels) [inline PlateSpace/Web/README.md]: by rules on a collection

Comment (edaniels) [inline PlateSpace/Web/README.md]: extra space between `the  Pipelines`

Comment (edaniels) [inline PlateSpace/Web/README.md]: Should be able to just say press new button.

File: PlateSpace/Web/README.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ In order to get our Stitch backend-as-a-service application fully set up, we mus
...
 
 #### geoNear pipeline
 
-Paginated geoLocation pipeline * in the app we are paginating from the nearest restaurant (nearest to the CURRENT_LOCATION value in src/mongodb-manager/mongodb-manager.js).
+Paginated geoLocation pipeline that allows the app to order and paginate restaurants from the restaurant that is closest to the `CURRENT_LOCATION` value as defined in *src/config.js*.
```
Comment (edaniels) [inline PlateSpace/Web/README.md]: A paginated

Comment (edaniels) [inline PlateSpace/Web/README.md]: that are closest

File: PlateSpace/Web/README.md [hunk 3]
```diff
@@ -330,7 +330,7 @@ The first step is to retrieve a valid Bearer token from Clarifai. To do so, sign
 * Sign in to the MongoDB Stitch console and press the `Add Service` button in the `Services` section of the left nav menu.
 * Select the __HTTP__ service, name it `Clarifai` and press the `Add service` button.
 * Select the __Rules__ tab and press the `Add Rule` button.
-* Check the `post` checkbox in the __Actions__ list
+* Enable the `Post` action in the __Actions__ list
```
Comment (edaniels) [inline PlateSpace/Web/README.md]: `post` I think is correct

Comment (rlondner) [inline PlateSpace/Web/README.md]: `Post` with capital P is what the new UI shows though...

File: PlateSpace/Web/README.md (hunk 1 above)
Comment (rlondner) [inline PlateSpace/Web/README.md]: fixed in https://github.com/mongodb/stitch-examples/pull/15/commits/a83b6c79275ffffae282ab74153c3f8bfc037cfd

File: PlateSpace/Web/README.md (hunk 2 above)
Comment (rlondner) [inline PlateSpace/Web/README.md]: fixed in https://github.com/mongodb/stitch-examples/pull/15/commits/70457eaa184a19daebc6e6f75b69b8d3d9967199

**Pull request merged**